MaterialDesignThemes.Wpf/RippleAssist.cs

```diff
@@ -54,6 +54,35 @@ public static bool GetIsCentered(DependencyObject element)
 
         #endregion
 
+        #region disable ripple
+
+        /// <summary>
+        /// Set to <c>True</c> to disable ripple effect
+        /// </summary>
+        public static readonly DependencyProperty IsRippleDisabledProperty = DependencyProperty.RegisterAttached(
+            "IsRippleDisabled", typeof(bool), typeof(RippleAssist), new FrameworkPropertyMetadata(false, FrameworkPropertyMetadataOptions.Inherits));
+
+        /// <summary>
+        /// Set to <c>True</c> to disable ripple effect
+        /// </summary>
+        /// <param name="element"></param>
+        /// <param name="value"></param>
+        public static void SetIsRippleDisabled(DependencyObject element, bool value)
```

Comment:
Can we drop the "Ripple";  we don't need the noun as we are already in the RippleAssist class, and it  keeps it inline with IsCentered.  So, "IsDisabled".